### DISCUSSION TITLE
multiple selection for replace preview when we use CTRL+H to replace in multiple files

### DISCUSSION THREAD

**Discussion by os3albert** [eclipse-platform/eclipse.platform.ui#631] [Ideas]
Add select or deselect multiple items once someone hits the replace button, inside preview mode i mean. inside of this view you can add also a checkbox that when it's selected, it can select all the items that is going to be affected by the replace action.

below I show the context of what I'm talking about, in general if you would like to select or deselect multiple items that's not possible in preview mode. Typically, multiple item selection is done using the shift key, similar to selecting files on a computer.

![image](https://user-images.githubusercontent.com/47365826/224580312-c81fea17-87d1-4e18-81d2-b164bca2c1cd.png)

![image](https://user-images.githubusercontent.com/47365826/224580323-a137c54c-222d-422a-ab30-fcbc8fd5c25a.png)

![image](https://user-images.githubusercontent.com/47365826/224580283-8434f7b9-6b28-4a85-b1be-8b58a897ee5b.png)

**Comment by mickaelistria** [comment]
Suggestion reported as #632.
